### DISCUSSION TITLE
Custom name revanced apps

### DISCUSSION THREAD

**Discussion by oldman20** [nikhilbadyal/docker-py-revanced#176] [General]
I cant find the way to custom revanced apps. How to do that? Thank!

**Comment by nikhilbadyal** [comment]
Unable to understand the query. Can you explain in little detail.

**Reply by oldman20** [reply]
I mean i want to set name of apk? Default is Revanced Extended,  I want to rename it to YouTube Extended?

**Reply by oldman20** [reply]
Example in https://github.com/j-hc/revanced-magisk-module
![image](https://github.com/nikhilbadyal/docker-py-revanced/assets/39668756/cd3372ae-227d-4a0f-be31-6f034f22d8b9)

**Reply by nikhilbadyal** [reply]
I don't think this is what you think of it. Have you tested if this changes the app name in launcher ?

**Reply by oldman20** [reply]
My stock Samsung OneUI Laucher cant change name of apps. I test that patch in https://github.com/j-hc/revanced-magisk-module forked and it worked!
